### ISSUE TITLE
one question for transductive SSL

### ISSUE THREAD

**Issue by songlu2333** [ahmetius/LP-DeepSSL#1]
In this paper, are the training set and the test set put together to construct a graph?

**Comment by ahmetius** [comment]
In our paper, we do not include the test set in the graph. We include the training set containing labeled and unlabeled examples. We use a transductive approach (label propagation) to train an inductive classifier, which generalizes to unseen test images.
